p2panda-js/src/session/session.ts

```diff
@@ -303,15 +303,17 @@ export class Session {
 
   /**
    * Signs and publishes an UPDATE operation for the given application data and
-   * matching schema. An UPDATE operation references the entry hash of the
-   * CREATE operation which is the root of this document.
+   * matching schema.
+   *
+   * The document to be updated is referenced by its document id, which is the
+   * operation id of that document's initial `CREATE` operation.
    *
    * Caches arguments for creating the next entry of this schema in the given
    * session.
    *
-   * @param documentId id of the document we update, this is the hash of the root `create` entry
+   * @param documentId id of the document we update, this is the id of the root `create` operation
    * @param fields application data to publish with the new entry, needs to match schema
-   * @param previousOperations array of operation hash ids identifying the tips of all currently un-merged branches in the document graph
+   * @param previousOperations array of operation ids identifying the tips of all currently un-merged branches in the document graph
```

Comment:
This kind of language is much nicer now! An operation is referenced by it's `OperationId`, rather than an unspecified `Hash` :+1:

Reply:
One onion-step less!